### DISCUSSION TITLE
gantt chart

### DISCUSSION THREAD

**Discussion by eosfor** [plotly/Plotly.NET#465] [General]
Hello everyone, 

I need a gantt chart, but i can't even imagine where to start :). What would be a best way to start experimenting?

**Comment by kMutagene** [comment]
Well, a gantt chart is basically a horizontal bar chart with a time axis, so this is what i'd start with. However, there are many tools out there to create this specific type of chart, so if Plotly.NET is not a hard requirement, i would use those. 

Feel free to contribute a respective preset though if you want to implement this in Plotly.NET ;)

**Reply by eosfor** [reply]
I wish i could, let me try to experiment. I'll try to share my results.
